WMF Framework/Theme.swift

```diff
@@ -169,580 +93,714 @@ public extension UIColor {
+            return .gray800
+        case .black:
+            return .gray700
+        case .widgetLight, .widgetDark:
+            return .clear
+        }
+    }
+
+    @objc public var chromeBackground: UIColor {
+        switch identifier {
+        case .light:
+            return .white
+        case .sepia:
+            return .beige100
+        case .dark:
+            return .gray700
+        case .black:
+            return .gray700
+        case .widgetLight, .widgetDark:
+            return .clear
+        }
+    }
+
+    @objc public var chromeShadow: UIColor {
+        switch identifier {
+        case .light, .widgetLight:
+            return .gray400
+        case .sepia:
+            return .taupe200
+        case .dark, .widgetDark:
+            return .gray650
+        case .black:
+            return .gray675
+        }
+    }
+
+    @objc public var overlayBackground: UIColor {
+        switch identifier {
+        case .light:
+            return .black.withAlphaComponent(0.5)
+        case .sepia:
+            return UIColor.taupe600.withAlphaComponent(0.6)
+        default:
+            return .black.withAlphaComponent(0.75)
+        }
+    }
+
+    @objc public var batchSelectionBackground: UIColor {
+        switch identifier {
+        case .light, .sepia, .widgetLight:
+            return .blue100
+        case .dark, .black, .widgetDark:
+            return .blue700
+
+        }
+    }
+
+    @objc public var referenceHighlightBackground: UIColor {
+        switch identifier {
+        case .light, .sepia, .dark, .widgetLight:
+            return .clear
+        case .black, .widgetDark:
+            return UIColor.white.withAlphaComponent(0.2)
+        }
+    }
+
+    @objc public var hintBackground: UIColor {
+        switch identifier {
+        case .light, .sepia, .widgetLight:
+            return .blue100
+        case .dark, .widgetDark:
+            return .gray800
+        case .black:
+            return .gray650
+        }
+    }
+
+    @objc public var hintWarningBackground: UIColor {
+        switch identifier {
+        case .light:
+            return .yellow600
+        case .sepia:
+            return .beige400
+        case .dark, .black:
+            return  .gray700
+        default:
+            return .clear
+        }
+    }
+
+    @objc public var animationBackground: UIColor {
+        switch identifier {
+        case .light, .sepia, .widgetLight:
+            return .gray150
+        case .dark, .black, .widgetDark:
+            return .gray700
+        }
+    }
+    
+    @objc public var overlayText: UIColor {
+        return .gray600
+    }
+    
+    @objc public var primaryText: UIColor {
+        switch identifier {
+        case .light, .sepia, .widgetLight:
+            return .gray700
+        case .dark, .black, .widgetDark:
+            return .gray100
+        }
+    }
+
+    @objc public var secondaryText: UIColor {
+        switch identifier {
+        case .light:
```

Comment:
I would go ahead and add `.widgetLight` to this line as well, looks like `WMFTodayContinueReadingWidgetViewController` still uses it.